.github/workflows/reusable-changesets-prerelease.yml

```
@@ -0,0 +1,75 @@
+name: "Prerelease Packages"
+on:
+  push:
+    branches:
+      - release/*
+  workflow_call:
+    inputs:
+      app-path:
+        description: "Specifies a base directory to run the task from"
+        required: false
+        default: './'
+        type: string
+      nodejs-version:
+        description: "Specifies which Node.js version to use"
+        required: false
+        default: '16'
+        type: string
+      package-manager:
+        description: "Specifies which package manager to use"
+        required: false
+        default: ''
+        type: string
+      build-command:
+        description: "Specifies a build command used on package finalizing before publish"
+        required: false
+        default: 'build'
+        type: string
+      prerelease-tag:
+        description: "Specifies a tag name that is used for prerelease mode"
+        required: false
+        default: 'pre'
+        type: string
+env:
+  NODE_ENV: production
+  CI: true
+
+jobs:
+  publish:
+    name: Run Changesets Pre release
```

p4: 요거 워딩 하나로 통일하면 어떨까요? pre-release, Pre release -> prerelease or Prerelease

```suggestion
    name: Run Changesets prerelease
```

결정되시면 아래도 적용 부탁드립니다!

---

Prerelease로 가시죠~